README.md

```diff
@@ -124,6 +124,39 @@ If you want to check what resources are going to be targeted without actually te
 cloud-nuke aws --resource-type ec2 --dry-run
 ```
 
+### Specify optional config file
+
+For more granularity, you can pass in a config file to specify which resources to terminate using regular expressions.
+
+```shell
+cloud-nuke --config path/to/file.yaml
```

Comment:
NIT: make sure this is updated to reflect what you now have in the description.

Reply:
Good catch!